cmd/package_managers.go

```diff
@@ -138,3 +202,115 @@ func fetchGitRepositoryFromRubyGems(packageName string, manager packageManagerCl
 	}
 	return v.SourceCodeURI, nil
 }
+
+// Gets the GitHub repository URL for the nuget package.
+func fetchGitRepositoryFromNuget(packageName string, manager packageManagerClient) (string, error) {
+	nugetIndexURL := "https://api.nuget.org/v3/index.json"
+	respIndex, err := manager.GetURI(nugetIndexURL)
+	if err != nil {
+		return "", sce.WithMessage(sce.ErrScorecardInternal, fmt.Sprintf("failed to get nuget index json: %v", err))
+	}
+	defer respIndex.Body.Close()
+	nugetIndexResults := &nugetIndexResults{}
+	err = json.NewDecoder(respIndex.Body).Decode(nugetIndexResults)
+
+	if err != nil {
+		return "", sce.WithMessage(sce.ErrScorecardInternal,
+			fmt.Sprintf("failed to parse nuget index json: %v", err))
+	}
+	nugetPackageBaseURL, err := getFieldFromIndexResults(nugetIndexResults.Resources,
+		"PackageBaseAddress/3.0.0")
+	if err != nil {
+		return "", err
+	}
+	nugetRegistrationBaseURL, err := getFieldFromIndexResults(nugetIndexResults.Resources,
+		"RegistrationsBaseUrl/3.4.0")
```

Comment:
```suggestion
		"RegistrationsBaseUrl/3.6.0")
```

You'll very likely want to include SemVer 2.0.0 packages. All the main NuGet experiences today include this.

Reply:
fixed